services/src/test/scala/cromwell/pipeline/service/impls/ProjectConfigurationServiceTestImpl.scala

```diff
@@ -19,7 +19,12 @@ class ProjectConfigurationServiceTestImpl(projectConfigurations: Seq[ProjectConf
   override def getLastByProjectId(projectId: ProjectId, userId: UserId): Future[ProjectConfiguration] =
     testMode match {
       case WithException(exc) => Future.failed(exc)
-      case _                  => Future.successful(projectConfigurations.head)
+
+      case _ =>
+        projectConfigurations.find(conf => conf.projectId == projectId && conf.active) match {
```

Comment:
That looks like a weird ad-hoc solution. Why do you need to filter by configuration activity? Can't you just not pass inactive configurations to this test implementation if you don't wan't them to appear in the result?

Reply:
Fixed